### DISCUSSION TITLE
Linking to campaigns and/or ads via ids possible ?

### DISCUSSION THREAD

**Discussion by M-Mommsen** [amzn/ads-advanced-tools-docs#14] [Q&A] [answered]
I would like to link the user directly to the page of a campaign or adgroup.
But the api and reports only give me a numeric ID. But a link seems to look like this:
https://advertising.amazon.de/cm/sb/campaigns/#CAMPAIGN#/targeting?entityId=ENTITY#PROFILE#
At first I though it might just be a base36 representation, but I don't get any matches.
Real life example:
campaignId 180902131375644 seems to be AS1AFXMW4KVNN , but base36 of the id would be 1s4hai9lb0.
(I also seems the first letter is always an A)
Is it a simple conversion, or do the entities have two different unique ids ?

**Comment by elizafarley** [comment, marked as answer]
Hi @M-Mommsen, the advertising console does use a different set of IDs compared to the API. Currently, there is not a way to resolve these two types of ID, so I don't think you will be able to reconstruct the console URL using data from the API.
